openshift/catalogd/manifests/14-deployment-openshift-catalogd-catalogd-controller-manager.yml

```diff
@@ -121,20 +120,21 @@ spec:
           secret:
             optional: false
             secretName: catalogserver-cert
```

Comment:
Should we add `items` here for `tls.crt` and `tls.key`?

Reply:
Unsure.

Reply:
Will look

Reply:
Done